library.js

```diff
@@ -0,0 +1,90 @@
+// declare empty array for library
+let myLibrary = [];
+let i = 0;
+//object constructor
+function Book(Title, Author, Pages, Read) {
+  this.Title = Title;
```

Comment:
use camelCase for naming variables and functions.
so this should be:
`this.title = title` and so on